.bc-exclude.php

```diff
@@ -128,5 +128,8 @@
 
         // Domain exceptions should not be extended in 3rd party code
         preg_quote('ADDED: Parameter domain was added to Method invalidDomain() of class Shopware\Core\System\SystemConfig\SystemConfigException', '/'),
+
+        // Type widening from string to ParsedRobots|string is backward compatible - all existing string usage continues to work
+        preg_quote('CHANGED: The parameter $rules of Shopware\Storefront\Page\Robots\Struct\DomainRuleStruct#__construct() changed from string to Shopware\Storefront\Page\Robots\Parser\ParsedRobots|string', '/'),
```

Comment:
should not be necessary anymore since https://github.com/shopware/shopware/pull/12941

Reply:
sadly not working